src/workflows/test_workflows/annotation/harmony_knn/script.py

```diff
@@ -0,0 +1,34 @@
+from mudata import read_h5mu
+import numpy as np
+import shutil
+import os
+import sys
+from pathlib import Path
+import pytest
+
+##VIASH START
+par = {
+    "input": "harmony_knn/output.h5mu"
+}
+
+meta = {
+    "resources_dir": "resources_test"
+}
+##VIASH END
+
+
+def test_run():
+    input_mudata = read_h5mu(par["input"])
+    expected_obsm = ["X_integrated_harmony", "X_leiden_harmony_umap"]
+    expected_obs = ["cell_type_pred", "cell_type_probability"]
+
+    assert "rna" in list(input_mudata.mod.keys()), "Input should contain rna modality."
+    assert all(key in list(input_mudata.mod["rna"].obsm) for key in expected_obsm), f"Input mod['rna'] obs columns should be: {expected_obsm}, found: {input_mudata.mod['rna'].obsm.keys()}."
+    assert all(key in list(input_mudata.mod["rna"].obs) for key in expected_obs), f"Input mod['rna'] obs columns should be: {expected_obs}, found: {input_mudata.mod['rna'].obs.keys()}."
```

Comment:
Would it be possible to add the contents of the newly created columns? (for example that the predictions are in fact labels and the probabilities are floats?)